### DISCUSSION TITLE
regexp in script: bug or misuse?

### DISCUSSION THREAD

**Discussion by M-Marton** [scandum/tintin#106] [Script help]
I feel it's a bug but find no other person talking about it, so probably just misuse of a the function. I can not get the found argument of regexp unless I type in the command.

```

#regexp {a b} {%1 %2} {#showme {&1};}{#showme unsuccesful;}
a

#ALIAS {regexp} NOW TRIGGERS {#regexp {a b} {%1 %2} {#showme {&1};}{#showme unsuccesful;}} @ {5}.
regexp

#ALIAS {regexp} NOW TRIGGERS {#regexp {a b} {%1 %2} {#showme {&1};}{#showme unsuccesful;};#showme DONE;} @ {5}.
regexp

DONE

#ALIAS {regexp} NOW TRIGGERS {#regexp {a b} {%1 %2} {#showme c;} {#showme d;};} @ {5}.
regexp
c
```

**Comment by scandum** [comment]
You need to use `#regexp {a b} {%%1 %%2}` or `#regexp {a b} {%* %*}` if you embed the regex in an alias.

Since #alias uses %1 %2 %3 as well.

**Reply by M-Marton** [reply]
thank you!
